crates/server/src/store/blocklist/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2022 Benjamin Fry <benjaminfry@me.com>
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// https://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// https://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+#![cfg(feature = "blocklist")]
+
+//! Blocklist resolver related types
+
+mod authority;
+mod config;
```

Comment:
Since the `config` module is short, I'd probably inline it here instead of splitting out into a separate file (acknowledging this is inconsistent with current style for now).